### PR TITLE
Darkmode

### LINKED ISSUE #31: Dark Mode
Dang this thing is bright

### LINKED ISSUE #37: Setting Page For Admin Personal Prefrences
See Title, will update this issue as new things are needed here

### PR DESCRIPTION
**Pull request by Simyon264**
### Your eyes no longer burn!
This PR resolves #31 and resolves #37. I had to update Bootstrap to version 5 for this PR and I am very unsure if I did it correctly.
Screenshots:
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by Geekyhobo** [comment]
Can you make sure to not overwrite the severity colors, I’m pretty sure they are defined in :root for css. I’ll hope on my computer and a minute and make an actual review in a moment.

File: SS14.Admin/Pages/Index.cshtml [hunk 1]
```diff
@@ -11,4 +11,5 @@
     <a class="btn btn-primary" asp-page="./Connections/Index">Connections</a>
     <a class="btn btn-primary" asp-page="./Logs/Index">Logs</a>
     <a class="btn btn-primary" asp-page="./Characters/Index">Characters</a>
+    <button type="button" class="btn btn-primary" onclick="toggleDarkMode()">Toggle darkmode</button>
```
Comment (Geekyhobo) [inline SS14.Admin/Pages/Index.cshtml:14]: Can you move this to a User settings page to the left of the Hello "your username here" use somthing like this icon to link the settings page <i class="fas fa-user-cog"></i> I have some more stuff I want to add to a setting page such as the PII censoring mechanic.

**Review by Geekyhobo — CHANGES_REQUESTED** [review]
Just move the darkmode button to a settings page, the rest of the page can be left blank bonus points if you just make the button the only  thing on the page and just center it in the middle of the screen

**Comment by Simyon264** [comment]
Reviews addressed

**Review by Geekyhobo — APPROVED** [review]
Lookas great!

File: SS14.Admin/Pages/Logs/Index.cshtml [hunk 2]
```diff
@@ -10,37 +10,41 @@
     var nextDisabled = (Model.Items.Count == 0) ? "disabled" : "";
     var nextPageIdx = Model.PageIndex + 1;
     var prevPageIdx = Model.PageIndex == 1 ? null : (int?) Model.PageIndex - 1;
+
+    var darkMode = Request.Cookies["darkMode"] == "true";
+    var darkModeClass = darkMode ? "dark" : "light";
+    var textClass = darkMode ? "text-light" : "text-dark";
```
Comment (PJB3005) [inline SS14.Admin/Pages/Logs/Index.cshtml:16]: You shouldn't need to change the HTML generated, this should be done by CSS.

File: SS14.Admin/Pages/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -3,10 +3,13 @@
 
 @{
     var signIn = SignInManager.GetSignInData(User);
+    var darkMode = Context.Request.Cookies["darkMode"] == "true";
+    var darkModeClass = darkMode ? "dark" : "light";
+    var textClass = darkMode ? "text-light" : "text-dark";
```
Comment (PJB3005) [inline SS14.Admin/Pages/Shared/_Layout.cshtml:8]: Ditto.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: SS14.Admin/Pages/Logs/Index.cshtml (hunk 2 above)
Comment (Simyon264) [reply SS14.Admin/Pages/Logs/Index.cshtml]: I think I did it like you wanted?

**Pull request merged**